pkgs/tools/X11/bumblebee/default.nix

```
@@ -132,6 +132,7 @@ in stdenv.mkDerivation rec {
   '';
 
   meta = with lib; {
+    broken = true; # At 03-08-2022
```

broken should be last in meta and please comment why it is marked borken

---

It errors when building. It is all I know. You can try building it and see for yourself...

`broken` starts with letter `b`, alphabetical order. The order you are trying to impose is very subjective.

---

meta is not sorted alphabetical.

```suggestion
    # linking fails with multiple error of type:
    # multiple definition of `bb_pm_method_string'; src/module.o:(.bss+0x0): first defined here
    broken = true;
```
not sure where to put the date

---

Done